### PR TITLE
Check if alignment files were removed from server before displaying their link

### LINKED ISSUE #3380: Old cases not showing alignments?
Seems old cases in Solna are not showing either a proper "no alignment file available" or the alignment files?
Cf e.g. https://scout.scilifelab.se/cust002/F0030513. Or F0025784, note especially the RNA part:
Screenshot 2022-06-07 at 16 00 36

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3380 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/app.py [hunk 1]
```diff
@@ -231,13 +230,6 @@ def count_cursor(pymongo_cursor):
         return len(list(cursor_copy))
 
 
-def register_tests(app):
```
Comment (northwestwitch) [inline scout/server/app.py:234]: This was used before to check if align files existed, then code was restructured and the check is no longer done

Comment (northwestwitch) [reply scout/server/app.py]: Ah no, I got confused. This code is used to check if reports exist. Reverting

Comment (dnil) [reply scout/server/app.py]: Eh, but we use that for all kinds of files, dont we?

Comment (northwestwitch) [reply scout/server/app.py]: Yes, you are right, I've realized the mistake and reverted

Comment (dnil) [reply scout/server/app.py]: Sorry for the lag - on a meeting and reading with one eye..

Comment (northwestwitch) [reply scout/server/app.py]: No problem it's still a draft after all!

**Comment by dnil** [comment]
Ah, and draft as well! Anyway, don't forget the splice junctions track!

**Comment by northwestwitch** [comment]
> Ah, and draft as well! Anyway, don't forget the splice junctions track!

No, right!

**Comment by northwestwitch** [comment]
Case page:

Before:

image


After:

image


Variant page:

Before:

image


After:

image

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Variants pagination after pressing "Filter variants" or "Clinical filter"
 - Layout of buttons Matchmaker submission panel (case page)
 - Removing cases from Matchmaker (simplified code and fixed functionality)
+- Removed buttons to display alignment files removed from server
```
Comment (dnil) [inline CHANGELOG.md:32]:
```suggestion
- Reintroduce check for missing alignment files purged from server
```

**Review by dnil — APPROVED** [review]

**Comment by northwestwitch** [comment]
Oh no, one of the the github actions is stuck in the process of installing the HTML 2 PDF renderer..

image

**Comment by dnil** [comment]
Worked for the others; restarting it will probably do the trick?

**Comment by northwestwitch** [comment]
> Worked for the others; restarting it will probably do the trick?

Tried, on this branch and on the new release one. Now they are both stuck :roll_eyes:

**Comment by northwestwitch** [comment]
Ah, it worked now, merging!

**Pull request merged**